targets/atlys_vga.py

```diff
@@ -0,0 +1,241 @@
+# Support for the Digilent Atlys board - digilentinc.com/atlys/
```

Comment:
I think this file needs to be updated for the latest changes in the other atlys targets? `_get_firmware_data` has been removed.